hedy.py

```diff
@@ -1966,7 +1971,15 @@ def find_indent_length(line):
     return number_of_spaces
 
 def needs_indentation(code):
-    keywords_requiring_indentation = ['if', 'als', 'si', 'for', 'repeat', 'répète', 'repete', 'herhaal']
+    en_keywords_requiring_indentation = ['if', 'for', 'repeat']
+    keywords_requiring_indentation = []
+    global en_commands
+    global lang_commands
+    for keyword in en_keywords_requiring_indentation:
+        if keyword in en_commands:
+            index = en_commands.index(keyword)
+            keywords_requiring_indentation.append(en_commands[index])
+            keywords_requiring_indentation.append(lang_commands[index])
```

Comment:
This only works if we can guarantee that the English and other language keywords are always in the same order. Is this the case? It might be nice to do some pre-processing somewhere and create a dictionary which contains these translations so that on runtime we can perform this action more efficiently.

For example:

```
# Create some efficient dict like this (English as key -> other language as value):
{'repeat': 'répète', ...}

keyword_requiring_indentation = en_keywords_requiring_indentation
keyword_requiring_indentation.append([k if look_up_dict.get(k, None) for k in en_keywords_requiring_indentation]
```

Reply:
Hi! I just did some improvements in #2493. Once this is merged you can easily include `KEYWORDS` from `hedy_content` to make the look-up easier. This is already a dictionary so you can simply iterate over it with the English keywords. Let me know if anything is unclear!